app/presenters/contact_presenter.rb

```diff
@@ -7,35 +7,45 @@ def initialize(contact)
     @contact = contact
   end
 
+  def content_id
+    contact.content_id
+  end
+
+  def update_type
+    "major"
+  end
+
   def present
     {
-      content_id: contact.content_id,
       title: contact.title,
       description: contact.description,
       format: "contact",
       locale: "en",
       publishing_app: "contacts",
       rendering_app: "contacts-frontend",
-      update_type: "major",
```

Comment:
Hmm so when I revert this change, I get a failing contract [test](https://github.com/alphagov/contacts-admin/blob/fd3a89c24d07e5baf2ab201353b3038cc6d365fa/spec/presenters/contact_presenter_spec.rb#L16-L19) against the V2 contacts schema. Does the above comment suggest that the V2 schemas will change in future so as to allow `update_type` as part of the payload again?